.eslintrc.es6.json

```diff
@@ -0,0 +1,42 @@
+{
+    "extends": "ovh",
+    "globals": {
+        "window": false,
```

Comment:
I will it try but it was already into the eslintrc.json files before I deleted it